roles/ceph-update/tasks/main.yml

```diff
@@ -8,69 +8,68 @@
   delay: 60
   delegate_to: "{{ groups['ceph_monitors'][0] }}"
 
-- name: stop ceph mon
-  service: name=ceph-mon-all state=stopped
-  when: "'ceph_monitors' in group_names"
+# tasks of ceph monitors
+- block:
```

Comment:
We need to protect from unnecessary stop of ceph mon and ceph osd when we are already on jewel. example use cases:
1: Cluster is already at 3.1 and on is re-running ursula
2: Upgrading from 3.1 to 3.1.1 and up